src/licensedcode/data/licenses/loop.LICENSE

```diff
@@ -0,0 +1,66 @@
+---
+key: loop
+short_name: Common Lisp LOOP License
+name: Common Lisp LOOP License
+category: Permissive
+owner: Symbolics Inc and MIT
+spdx_license_key: LOOP
+other_urls:
+    - https://gitlab.com/embeddable-common-lisp/ecl/-/blob/develop/src/lsp/loop.lsp
+    - http://git.savannah.gnu.org/cgit/gcl.git/tree/gcl/lsp/gcl_loop.lsp?h=Version_2_6_13pre
+    - https://sourceforge.net/p/sbcl/sbcl/ci/master/tree/src/code/loop.lisp
+    - https://github.com/cl-adams/adams/blob/master/LICENSE.md
+    - https://github.com/blakemcbride/eclipse-lisp/blob/master/lisp/loop.lisp
+    - https://gitlab.common-lisp.net/cmucl/cmucl/-/blob/master/src/code/loop.lisp
+ignorable_copyrights:
+    - Copyright (c) 1986 by the Massachusetts Institute of Technology
+    - Copyright (c) 1989, 1990, 1991, 1992 by Symbolics, Inc.
+ignorable_holders:
+    - Symbolics, Inc.
+    - the Massachusetts Institute of Technology
```

Comment:
add a minimum_coverage: 95 and two new rules, one for each text pointing to mit-old-style-no-advert with a relevance of 99.